model.md

```diff
@@ -0,0 +1,164 @@
+# Model
+
+### <a name="struct"></a> Struct
+
+This is the primary object...
+
+| Field   | Type                      | Required | Notes                                                  |
+|---------|---------------------------|----------|--------------------------------------------------------|
+| name    | ref                       | true     | The common name used to identity the type of structure |
+| label   | string or [Label](#label) | true     | Describes the struct in the UI                         |
+| collectionLabel | string or [Label](#label) | true | Describes a collection of the struct in the UI     |
+| fields  | [Field](#field)[]         | true     |                                                        |
+| blocks  | [block](#block)[]         | true     | Array of [blocks](#block)                              |
```

Comment:
For consistency `[block]` should probably be `[Block]`

Reply:
Agreed.